### PR TITLE
Fix ember-cli commands to work with windows

### PR DESCRIPTION
**Pull request by cobbspur**

### CODE REVIEW

**Comment by novaugust** [comment]
SORRY DAVID AND FELIX

File: Gruntfile.js [hunk 1]
```diff
@@ -278,16 +279,16 @@ var _              = require('lodash'),
...
-                                return './node_modules/.bin/ember build --silent';
+                                return emberPath + ' build --silent';
 
                             case 'test':
-                                return './node_modules/.bin/ember test --silent';
+                                return emberPath + ' test --silent';
 
                             default:
-                                return './node_modules/.bin/ember build --silent';
+                                return emberPath + ' build --silent';
```
Comment (novaugust) [inline Gruntfile.js]: Hey actually, want to just delete this default stanza? it's dead code that isn't used anywhere

**Comment by novaugust** [comment]
I was also lazy about path resolution [here](https://github.com/TryGhost/Ghost/blob/master/core/client/lib/asset-delivery/index.js#L6-L7) with ember-cli

**Comment by cobbspur** [comment]
done -iddly-un :)

**Comment by novaugust** [comment]
what about that second file?  see my last comment :D

**Comment by ErisDS** [comment]
@novaugust that bit works fine it seems!

**Comment by novaugust** [comment]
huh, okay then!

**Pull request merged**